packages/core/src/lib/toggle-buttons.svelte

```diff
@@ -51,7 +51,6 @@ $: getButtonClasses = (option: string) => {
 };
 
 const handleClick = (value: string) => {
-  selected = value;
```

Comment:
I wonder if the caller could also put `selected` in a store and then when they handle the input event, if they could just set $selected back to the original value (which IIRC would still trigger an update to this component even though the value is the same).

It might flash, but I think the transition would be quick.

Reply:
Also an option, but this behavior is weird and I think the component should be purely presentational instead of mixing stateful concerns.